manifests/database/postgresql.pp

```diff
@@ -9,12 +9,14 @@
   $manage_server        = $puppetdb::params::manage_dbserver,
   $manage_package_repo  = $puppetdb::params::manage_pg_repo,
   $postgres_version     = $puppetdb::params::postgres_version,
+  $postgres_datadir     = $puppetdb::params::postgres_datadir,
```

Comment:
I have not found another way to change the postgresql datadir in case if i use puppetlabs-puppetdb module.
If i use class puppetdb, as stated in the [documentation](https://docs.puppet.com/puppetdb/4.3/install_via_module.html#step-2-assign-classes-to-nodes)
```
class { 'puppetdb':
    listen_address => 'example.foo.com'
}
```

, and i trying to use class postgresql, i get error "cannot redeclare class" or something like that. At this moment I am not at my workplace, and i can not provide the exact text of the error.